<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuwidget</class>
 <widget class="QWidget" name="menuwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>160</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>menuwidget</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="toolButton_cameraDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>138</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/xiala_03.png);}
QToolButton:hover{border-image: url(:/xiala_03_02.png);}
QToolButton:pressed{border-image: url(:/xiala_03_02.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_left_nav</string>
   </attribute>
  </widget>
  <widget class="QToolButton" name="toolButton_workingState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>138</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/xiala_02.png);}
QToolButton:hover{border-image: url(:/xiala_02_02.png);}
QToolButton:pressed{border-image: url(:/xiala_02_02.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_left_nav</string>
   </attribute>
  </widget>
  <widget class="QToolButton" name="toolButton_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/caidan_01.bmp);}

QToolButton:hover{border-image: url(:/caidan_02.bmp);}

QToolButton:pressed{border-image: url(:/caidan_02.bmp);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_startDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>138</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/xiala_06.png);}
QToolButton:hover{border-image: url(:/xiala_06_02.png);}
QToolButton:pressed{border-image: url(:/xiala_06_02.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_left_nav</string>
   </attribute>
  </widget>
  <widget class="QToolButton" name="toolButton_systemState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>138</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/xiala_01.png);}
QToolButton:hover{border-image: url(:/xiala_01_02.png);}
QToolButton:pressed{border-image: url(:/xiala_01_02.png);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_left_nav</string>
   </attribute>
  </widget>
  <widget class="QToolButton" name="toolButton_mapNavigation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>138</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/xiala_04.png);}
QToolButton:hover{border-image: url(:/xiala_04_02.png);}
QToolButton:pressed{border-image: url(:/xiala_04_02.png);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_left_nav</string>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_left_nav"/>
 </buttongroups>
</ui>
